docs/measurements/procedures/rigid_waist_shift.md

```diff
@@ -12,10 +12,68 @@
       After global corrections are done and trimmed in the machine, one applies a rigid waist shift in a given IR and scans the colinearity knob for the value that minimizes the $|C^{-}|$.
       These settings, when taking away the rigid waist shift, will minimize local coupling and its impact at the IP.
```

Comment:
Some more details, why this waist-shift is needed maybe? 
Something in like: due to phase advances in IR local coupling is compensated within the IR and does not show up in global measurement, waist shift breaks this symmetry and coupling leaks out of IR.

What is the coliniearity knob? Why does it minimize |C-|?

Reply:
This is written in the "quick info about the procedure" admonition at the top of the page. For the last two questions, you'd have to read at least my IPAC papers, and I don't want to make this page kilometers long!

Reply:
put references to your IPAC papers then ;)

Reply:
Done in [25e8b67](https://github.com/pylhc/pylhc.github.io/pull/74/commits/25e8b67700691db7530d4580984a2128b0e21416)